src/cljs/athens/devcards/all_pages.cljs

```diff
@@ -51,37 +55,82 @@
     (.toLocaleString  (js/Date. x))))
 
 
+(def table-cell-background-color-hover
+  (opacify (:panel-color HSL-COLORS) (first OPACITIES)))
```

Comment:
I'd like to come up with a cleaner way to do colors and opacities down the line. Ideally, we don't need to import too many constants, and can use more `+style` functions.